sway/commands/bind.c

```diff
@@ -69,22 +69,24 @@ bool binding_key_compare(struct sway_binding *binding_a,
 	return true;
 }
 
-struct cmd_results *cmd_bindsym(int argc, char **argv) {
+static struct cmd_results * cmd_bindsym_or_bindcode(int argc, char **argv, bool bindcode) {
+	const char* bindtype = bindcode ? "bindcode" : "bindsym";
```

Comment:
Incorrect *

Reply:
Clarification: the correct style is `const char *bindtype...`